### PR TITLE
Added missing parameters required to run program on the sample data

### PR DESCRIPTION
**Pull request by isaacaka**
Added missing parameters (first and last years for the ERA product) required to run program on the US-ARc sample data

### CODE REVIEW

**Comment by dorchard** [comment]
Closes https://github.com/Cambridge-ICCS/ONEFlux/issues/13

File: README.md [hunk 1]
```diff
@@ -107,27 +107,31 @@ Run all steps in the pipeline:
...
 ```
-python runoneflux.py all "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --mcr ~/bin/matlab/v94/ --recint hh
+python runoneflux.py all "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --mcr ~/bin/matlab/v94/ --recint hh --era-fy 1989 --era-ly 2014
 ```
 
 
 Run nighttime partitioning method:
 ```
-python runoneflux.py partition_nt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log
+python runoneflux.py partition_nt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --era-fy 1989 --era-ly 2014
```
Comment (dorchard) [inline README.md:119]: --era-fy and --era-ly not needed here.

File: README.md [hunk 2]
```diff
@@ -107,27 +107,31 @@ Run all steps in the pipeline:
...
 ```
 
 Run daytime partitioning with only single percentile and/or a single USTAR threshold type data product (recommended for first executions), use:
 - *--prod y*: processes only VUT USTAR threshold product
 - *--perc 50*: processes only 50-percentile USTAR threshold
 - *--force-py*: forces execution of Python partitioning code (replaces existing outputs)
 ```
-python runoneflux.py partition_dt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --prod y --perc 50 --force-py
+python runoneflux.py partition_dt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --prod y --perc 50 --force-py --era-fy 1989 --era-ly 2014
```
Comment (dorchard) [inline README.md:127]: --era-fy and --era-ly not needed here.

File: README.md [hunk 3]
```diff
@@ -107,27 +107,31 @@ Run all steps in the pipeline:
...
 ```
 
 Note that for the execution of the partitioning steps, only if the output ```*.csv``` file doesn’t exist (e.g., ```nee_y_50_US-ARc_2005.csv```), the code will run and generate the file. If it exists, nothing will be done (unless the flag --force-py is used).
 
+Issues Related to Running Sample US-ARc
 
+- https://github.com/fluxnet/ONEFlux/issues/39
```
Comment (dorchard) [inline README.md:134]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -107,27 +107,31 @@ Run all steps in the pipeline:
...
 - *--perc 50*: processes only 50-percentile USTAR threshold
 - *--force-py*: forces execution of Python partitioning code (replaces existing outputs)
 ```
-python runoneflux.py partition_dt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --prod y --perc 50 --force-py
+python runoneflux.py partition_dt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --prod y --perc 50 --force-py --era-fy 1989 --era-ly 2014
 ```
 
 Note that for the execution of the partitioning steps, only if the output ```*.csv``` file doesn’t exist (e.g., ```nee_y_50_US-ARc_2005.csv```), the code will run and generate the file. If it exists, nothing will be done (unless the flag --force-py is used).
 
+Issues Related to Running Sample US-ARc
```
Comment (dorchard) [inline README.md:132]:
```suggestion
```

**Review by dorchard — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (isaacaka) [inline README.md:119]:
```suggestion
python runoneflux.py partition_nt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log
```

File: README.md (hunk 2 above)
Comment (isaacaka) [inline README.md:127]:
```suggestion
python runoneflux.py partition_dt "../datadir/" US-ARc "US-ARc_sample_input" 2005 2006 -l fluxnet_pipeline_US-ARc.log --prod y --perc 50 --force-py
```

**Review by dorchard — APPROVED** [review]
Looks good to me if @gilbertozp is happy with this.

**Review by gilbertozp — APPROVED** [review]
Added parameters and documentation look good

**Pull request merged**